src/styles/Breadcrumps.scss

```
@@ -0,0 +1,11 @@
+.breadcrumps{
+    text-decoration:none;
+    color: white;
+  
+  
+  
+    &--unused{
+      color:gray;
+    }
+
```

What happened to the line breaks 😂 
```suggestion
.breadcrumps {
  text-decoration:none;
  color: white;
  
  &--unused{
    color:gray;
  }
```

---

changed